clean-abap/CleanABAP.md

```diff
@@ -3922,38 +3921,19 @@ DATA:
   ,e TYPE f.
 ```
 
-### Use the ABAP Formatter before activating
-
-> [Clean ABAP](#clean-abap) > [Content](#content) > [Formatting](#formatting) > [This section](#use-the-abap-formatter-before-activating)
+### Use an Automated Code Formatting Tool
 
-Apply the ABAP Formatter - Shift+F1 in SE80, SE24, and ADT - before activating an object.  
-Note: ABAP Formatter is known as Pretty Printer in SAP GUI.
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Formatting](#formatting) > [This section](#use-an-automated-code-formatting-tool)
 
-If you modify a larger unformatted legacy code base,
-you may want to apply the ABAP Formatter only to selected lines
-to avoid huge change lists and transport dependencies.
-Consider formatting the complete development object
-in a separate Transport Request or Note.
+Use an automated code formatting tool with a consistent setting in your team.
 
 > Read more in _Chapter 5: Formatting: Team Rules_ of [Robert C. Martin's _Clean Code_].
 
-### Use your team's ABAP Formatter settings
-
-> [Clean ABAP](#clean-abap) > [Content](#content) > [Formatting](#formatting) > [This section](#use-your-teams-abap-formatter-settings)
+### Use the ABAPCleaner
 
-Always use your team's ABAP Formatter settings.
-Specify them under
-* Eclipse: Right-click on the project in the Project Explorer > _Properties_ > _ABAP Development_ > _Editors_ > _Source Code Editors_ > _ABAP Formatter_
-* Eclipse (alternative navigation): _Menu_ > _Window_ > _Preferences_ > _ABAP Development_ > _Editors_ > _Source Code Editors_ > Click on the _ABAP Formatter_ link on the right-hand side > Select project in pop-up
-* SAP GUI: _Menu_ > _Utilities_ > _Settings ..._ > _ABAP Editor_ > _Pretty Printer_.
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Formatting](#formatting) > [This section](#use-the-abap-cleaner)
 
-Set _Indent_ and _Convert Uppercase/Lowercase_ > _Uppercase Keyword_
-as agreed in your team.
-
-> [Upper vs. Lower Case](sub-sections/UpperVsLowerCase.md) explains
-> why we do not give clear guidance for the type case of keywords.
->
-> Read more in _Chapter 5: Formatting: Team Rules_ of [Robert C. Martin's _Clean Code_].
+Always use the [ABAPCleaner](https://github.com/SAP/abap-cleaner), ideally with the default configuration.
```

Comment:
I think this should be the same as before: Always use your team's formatter settings. 😉